doc/protocols/sigprefixlist/SigPrefixListProtocol.md

```diff
@@ -0,0 +1,71 @@
+# SIG Prefix List Protocol
+
+## Overview
+The protocol is designed to enable SIGs to automatically fetch IP prefix to AS ID mappings. It uses a distributed coordinated infrastructure with the following design and security goals. 
+
+The system should
+- Be reselient to hijacking attacks 
+- Be resilient to flooding attacks on SIGs
+- Be resilient to downgrade attacks to IP internet where SCION connections are possible
+- Not be a single point of failure
+
+To acheive the above properties the following services are used
+- Mapping Service (MS)
+- SCION Internet Gateway (SIG)
+
+- Publishing Infrastructure Services
+    - Publishing List Node (PLN)
+    - Publishing Consensus Node (PCN)
+
+## Services 
+### SIG
+The SIG can perform the following actions:
+
+- Get mapping from Mapping Service:
+    - A SIG that requires a mapping from an ASID to an IP prefix queries the Mapping Service in CoreASs. 
+- Add mapping 
+    - To add a mapping for the AS that the SIG is deployed it, it submits the mapping to a Mapping Service in the CoreASs.
+
+### Mapping Service (MS)
+A MS should be deployed in atleast one Core AS of an ISD that supports the use of SIGs.
+
+The Mapping services performs the following actions: 
+- Submit lists of mappings to Publishing Infrastructure  
+    - The MS assimilates entries from different SIGs to submit to the Publishing Infrastructure. 
+- Reply to SIG mapping queries 
+    - The MS responds to SIG mapping queries with IP prefix - IA mappings 
```

Comment:
>IA mappings 

Be consistent, either use `AS ID mappings` or `IA mappings` (short for ISD-AS identifiers mappings)

Reply:
I changed them to IA mappings

Reply:
```suggestion
    - The MS responds to SIG mapping queries with IP prefix--IA mappings.
```